### DISCUSSION TITLE
how to render only some fields and UI elements on template

### DISCUSSION THREAD

**Discussion by vnali** [craftcms/cms#9693] [Q&A] [answered]
Hi.
before Craft 3.5, I was able to render some fields with:
```
{% include '_includes/fields' with {
fields: selectedFieldsOfLayout,
element: element,
static: false,
} only %}
```
after Craft 3.5 and custom UI enhancement, this [KB](https://craftcms.com/knowledge-base/upgrading-to-craft-3-5) says we can render custom fields and UI elements with this lines:
```
$form = $myElement->getFieldLayout()->createForm($myElement); //
$fieldsHtml = $form->render();
```
and passing $fieldHtml to twig. but it render all custom fields and UI elements.

i know ​second argument of createForm() ,pass static status, but now I am wondering how can i pass only selected custom fields and UI elements to twig? is it related to third argument of createForm()?

**Comment by brandonkelly** [comment, marked as answer]
You could technically still render your filtered list of fields the old way; you just wouldn’t get any UI elements (Template, `<hr>`, etc.) defined for the field layout.

If you want to use the field layout’s `render()` method, it’s all-or-nothing.
